cv32e40s/bsp/crt0.S

```diff
@@ -23,8 +24,13 @@ _start:
 /* initialize global pointer */
 .option push
 .option norelax
-1:	auipc gp, %pcrel_hi(__global_pointer$)
+1:auipc gp, %pcrel_hi(__global_pointer$)
```

Comment:
This instruction is not aligned and the lines below use a mix of tabs and spaces.

I am ok with this. But if other changes are made then this could preferably be fixed at the same time.

Reply:
I will get this fixed